docs/DeveloperGuide.md

```diff
@@ -1,38 +1,235 @@
+
+
+The user will then be presented with a menu showing the different user details he can view as seen in the code
+snippet below.  
+
+````
+View user settings
+1. View Name
+2. View Weight
+3. View Height
+4. View Age      
+5. View Gender
+6. View Daily Caloric limit
+7. View Calories left today
+8. Back
+````
+
+Step 2. The user chooses to view his weight by inputting the number `2` to choose `View Weight`.
+This calls the getter method `getWeight()` in the entity `User` to return the current weight of the user
+and initializes the variable `weight` with that value. 
+
+<img src="uml/ViewUserCommandSD2.png" alt="Sequence Diagram 2" width="500">
+
+It then prints to screen the weight of the user like in the code snippet below.
+
+````
+Weight: {weight} kg
+
+Continue viewing?
+1. Yes
+2. No
+````
+Step 3. The user chooses to continue viewing by inputting the number `1` and chooses `View Daily Caloric Limit` next
+by inputting the number `6`. This calls the getter method `getCaloricLimit()` in the entity `User` to return the
+current daily caloric limit of the user and initializes the variable `caloricLimit` with that value. 
+
+<img src="uml/ViewUserCommandSD3.png" alt="Sequence Diagram 2" width="500">
+
+It then prints to screen the daily caloric limit of the user like in the code snippet below.
+
+````
+This is your daily caloric limit: 
+{caloricLimit} Kcal
+
+Continue viewing?
+1. Yes
+2. No
+````
+
+Step 4. The user executes the command `update` to update his user details. The user is presented with a menu showing the
+different user details he can update as seen in the code snippet below.
+
+````
+Update user settings
+1. Update Name
+2. Update Weight
+3. Update Height
+4. Update Age
+5. Update Gender
+````
+The user chooses to update his weight by inputting the number `2`. This causes the state of the user's details to be 
+modified and the user's weight as well as his caloric limit will be updated in accordance to the new weight entered.
+
+> Insert UML diagram showing the update process
+
+Step 5. The user then executes the command `view` to view his updated weight and daily caloric limits.
+# Appendix: Requirements
+
+## Product scope
+
+
+### Target user profile:
+
+- Is an NUS student
+- Looking to lose/maintain their weight
+- Looking to make healthier choices in terms of food
+- Can type fast
+- Prefers typing to mouse interactions
+- Is reasonably comfortable using CLI apps
+
+### Value proposition:
+For individuals trying to lose weight, managing their daily caloric intake is crucial. However, many may find it 
+hard to track and manage their calories. _LifeTracker_ allows users to easily automate the tracking of their calories
+and keep a record of their daily caloric intake. 
+
+_LifeTracker_ can also keep track of the exercises that users have done and factor it in to their daily calories lost. This allows
+the user to keep track of their net calorie gain on a daily basis.
+
+### User Stories
```

Comment:
a bug with your user stories table, it does not show up as intended on the DG website

<img width="814" alt="image" src="https://user-images.githubusercontent.com/88079214/228434283-65bf8379-5230-44f0-a6dc-fb7bc5da43c3.png">